packages/react-core/src/components/Alert/AlertIcon.js

```diff
@@ -30,7 +30,7 @@ const AlertIcon = ({ variant, className, ...props }) => {
   const Icon = variantIcons[variant];
   return (
     <div {...props} className={css(styles.alertIcon, className)}>
-      <Icon size="md" />
+      <Icon />
```

Comment:
Core html/css sets 1.5rem on the icon class.  using ems here makes this (1.5rem * 1.5em) making the icon 36px instead of 24px.  Setting to 1rem fixes this and still allows core to drive the size of the icon inside of the alert.